main.cpp

```diff
@@ -126,384 +167,529 @@ struct CarWash
     float configuredFrequencyComms = 60.00f;
-    // Current total power consumption of communications electronics
     float netPowerConsumption = 30.0f;
 
     struct DetachmentPersonnel
     {
-        // Rank of personnel
         std::string rank = "CPL";
-        // Is qualified to man communications
         bool qualifiedToManComms = false;
-        // Number of experienced missions
         int numberOfExperiencedMissions = 3;
-        // Hours of rations left
         int hoursRationsLeft = 12;
-        // Is well enough to continue mission
         bool wellEnoughToContinue = true;
 
-        // Shift for sentry duty
         void shiftSentry(int sentryDuration = 8, bool isWell = true);
-        // Prepare for artillery attack
         void prepareForArtilleryAttack(int coverDuration = 40, bool isShellscrapePresent = false);
-        // Consume rations
         void consumeRations(int caloriesConsumed = 1000);
     };
 
-    // Change radio frequency to facilitate comms
     void changeRadioFrequency(float targetFrequency = 60.00f);
-    // Deactivate comms during standing orders in event of radio silence
     void deactivateComms(bool isRadioSilence = true, bool isDangerPresent = false);
-    // Jack up antenna to specified height
     void jackUpAntenna (bool isVehicleGrounded = true, int unitsToJackUp = 8);
 };
-/*
-Thing 5) Barrel
-5 properties:
-    1) Length (inches) (float)
-    2) Bullet diameter (mm) (float)
-    3) Has adapter for muzzle attachment (bool)
-    4) Barrel material (std::string)
-    5) Barrel threading (mm) (int)
-3 things it can do:
-    1) Impart spin to bullet
-    2) Guide bullet along specified vector
-    3) Contain expanding gases to accelerate bullet
- */
+
+void MilitaryTrunkCommsVehicle::DetachmentPersonnel::shiftSentry(int sentryDuration, bool isWell)
+{
+    std::string orderSendSentry = "no";
+    if (isWell == true)
+    {
+        std::cout << "Sentry: shift activated for next " << sentryDuration << " hours.\n";
+    }
+    else
+    {
+        std::cout << "Soldier not well enough to carry out sentry duty!";
+    }
+    return;
+}
+
+void MilitaryTrunkCommsVehicle::DetachmentPersonnel::prepareForArtilleryAttack(int duration, bool isShellscrapePresent)
+{
+    if(isShellscrapePresent == true)
+    {
+        std::cout << "Taking cover in shellscrape for " << duration << " minutes\n";
+    }
+    else
+    {
+        std::cout << "Taking cover on ground for " << duration << " minutes\n";
+    }
+    return;
+}
+
+void MilitaryTrunkCommsVehicle::DetachmentPersonnel::consumeRations(int caloriesConsumed)
+{
+    std::cout << "Recovered " << caloriesConsumed << " calories for soldiers on field\n";
+    return;
+}
+
+void MilitaryTrunkCommsVehicle::changeRadioFrequency(float targetFrequency)
+{
+    std::cout << "Changing communications frequency to " << targetFrequency << " MHz, refighting link\n";
+    return;
+}
```

Comment:
Functions that don't return anything don't need a `return;` statement
- [ ] revise